docs/examples/15_minutes_to_QCoDeS.ipynb

```diff
@@ -767,13 +809,9 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Qcodes `Measurement` module provides a context manager for registering parameters to measure and store results. The measurement is first linked to the correct experiment and station by passing them as an argument. If no argument is given, the currently loaded experiment and station are taken as default. \n",
-    "\n",
-    "Qcodes is capable of storing relation between parameter i.e. which parameter is independent and which parameter depends on another one. This capability is later used to make useful plots, where the knowledge of interdependencies is used to infer what goes on x-axis and what on y-axis. All the parameters required in the measurement are first registered. In case a parameter depends on another parameter then another parameter is declared as a 'setpoint'. In Qcodes lingo 'setpoint' parameter means the parameter on which another parameter depends. In current example, dac.ch1 is the independent parameter and dmm.v1 is the dependent parameter whose setpoint is dac.ch1.\n",
-    "\n",
-    "`meas.write_period` is used to define the periods after which the data is committed to the database. We do not commit individual datapoint during measurement to the database but only after some amount of data is collected in stipulated time period (in this case for 2 seconds). The default value of write_period is 5 seconds.   \n",
+    "Qcodes `Measurement` module provides a context manager for registering parameters to measure and store results. The measurement is first linked to the correct experiment and to the station by passing them as arguments. If no arguments are given, the currently loaded experiment and station are taken as defaults. \n",
```

Comment:
we don't have yet a concept of an active experiemnt in qcodes. so it actually takes an experiment with that latest timestamp from the database.
```suggestion
    "Qcodes `Measurement` module provides a context manager for registering parameters to measure and store results. The measurement is first linked to the correct experiment and to the station by passing them as arguments. If no arguments are given, the "latest" experiment and station are taken as defaults. \n",
```